### PR TITLE
support activating more than 2 pdks

### PR DESCRIPTION
**Pull request by joamatab**
- deprecate Pdk.base_pdk in favor of Pdk.pdks
- add logic to activate multiple pdks
- add docs

@tvt173 
@kithminrw

### CODE REVIEW

**Comment by kithminrw** [comment]
If I submit multiple PDKs in a list as pdks=[ubcpdk, ctpdk], then it should work right? Ill try this after it is merged.

File: gdsfactory/pdk.py [hunk 1]
```diff
@@ -229,6 +229,7 @@ class Pdk(BaseModel):
         default=floorplan_with_block_letters, exclude=True
     )
     base_pdk: Pdk | None = None
+    pdks: list[Pdk] = Field(default_factory=list)
```
Comment (tvt173) [inline gdsfactory/pdk.py:232]: confusing name, no? maybe `base_pdks`?

**Review by tvt173 — CHANGES_REQUESTED** [review]
I suppose it's ok, but I could foresee some confusing situations if you base off multiple PDKs and they don't all play nicely together...

**Comment by joamatab** [comment]
I think some fabs have pdks done by multiple companies with different cells, happy to merge and make a new release

File: gdsfactory/pdk.py (hunk 1 above)
Comment (joamatab) [reply gdsfactory/pdk.py:232]: Yes, good point

**Review by tvt173 — APPROVED** [review]

**Pull request merged**